source/Server/Decompilation/ILDecompiler.cs

```diff
@@ -23,6 +23,13 @@ public class ILDecompiler : IDecompiler {
             Argument.NotNull(nameof(streams), streams);
             Argument.NotNull(nameof(codeWriter), codeWriter);
 
+            // Write header

```

Comment:
There is a separate copy of this code in .NET Framework, we'll need to update there as well.
(I don't _always_ update it, but this seems small enough and you already updated tests there)

Reply:
Manual header writing has been removed.